archinstall/lib/installer.py

```diff
@@ -52,7 +52,12 @@
 from .storage import storage
 
 # Any package that the Installer() is responsible for (optional and the default ones)
-__packages__ = ['base', 'base-devel', 'linux-firmware', 'linux', 'linux-lts', 'linux-zen', 'linux-hardened']
+__packages__ = ['base', 'sudo', 'linux-firmware', 'linux', 'linux-lts', 'linux-zen', 'linux-hardened']
+
+# base-devel is actually more a developer's thing and isn't strictly required in arch systems
```

Comment:
I don't think we need this comment here, it's enough if it's on the PR and we can find it through the commit